fv/bpf_test.go

```diff
@@ -2270,6 +2271,48 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 								}
 							})
 
+							if testOpts.protocol == "tcp" {
```

Comment:
Maybe we should define 
```
ConditionallyIt(testOpts.protocol == "tcp", "should survive conntrack cleanup sweep", func() {
  ...
})
```
To replace `if ... { It(...` with the extra nesting

Reply:
I like that, I will do it!

Reply:
However, it is a bit disappointing that Go is no C and has no preprocessor :-/

It would have to look like 

```
ConditionallyIt(func() bool {return testOpts.protocol == "tcp")}, "should survive conntrack cleanup sweep", func() {
  ...
})
```

it may not be too bad